modules/oadp-using-ca-certificates-with-velero-command.adoc

```diff
@@ -0,0 +1,77 @@
+
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.adoc
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-azure.adoc
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-gcp.adoc
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-mcg.adoc
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-using-ca-certificates-with-velero-command-aliased-for-velero-deployment_{context}"]
+== Using CA certificates with the velero command aliased for Velero deployment
```

Comment:
```suggestion
= Using CA certificates with the velero command alias
```

The first heading should be H1
The heading can be shortened and the intro line can have those details -- Up to you